### ISSUE TITLE
GU0082 should not warn here.

### ISSUE THREAD

**Issue by JohanLarsson** [GuOrg/Gu.Analyzers#190]
```cs
[TestCase(double.MinValue)]
[TestCase(int.MinValue)]
public void WithDouble(double value)
{
}
```
